guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -43,11 +43,9 @@ For example, in one of the <a href="https://github.com/magento/magento2/blob/2.0
 
 The root source files for the Blank theme:
 
-<ul>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-m.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/styles-m.less</a></code></li>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-l.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/styles-l.less</a></code></li>
-<li><code><a href="{{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/print.less" target="_blank">&lt;Magento_Blank_theme_dir&gt;/web/css/print.less</a></code></li>
-</ul>
+*   `[<Magento_Blank_theme_dir>/web/css/styles-m.less]({{ site.mage2000url }}app/design/frontend/Magento/blank/web/css/styles-m.less)`
```

Comment:
Please also revert this. Markdown code is not supported **within** HTML tables. Within HTML tables you must use HTML code only.